### PR TITLE
Site: add docs on Kubernetes troubleshooting

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]
Nice doc addition :+1:

**Review by snazy — DISMISSED** [review]

File: site/docs/guides/kubernetes.md [hunk 1]
```diff
@@ -332,3 +332,153 @@ To uninstall the Helm chart and delete the `nessie` release from the `nessie-ns`
...
+cp /root/.bashrc /home/debug/ && chown 185:0 /home/debug/.bashrc 
+su - debug
+```
+
+After running the above commands, you should be able to use `jps`, `jcmd`, `jmap`, etc. as well as
+other tools like `async-profiler`, `jfr` (Java Flight Recorder), etc. For example, you can check the
+Nessie server's JVM threads with `jcmd`, or run `async-profiler` to profile the Nessie server:
+
+```bash
+async-profiler -d 30 -f profile.html 1
```
Comment (adutra) [inline site/docs/guides/kubernetes.md:455]: Sorry for dismissing your approvals, I'm trying to validate this example but async-profiler seems buggy on an arm64 machine. Switching to amd64 to see if it's better.

Comment (adutra) [reply site/docs/guides/kubernetes.md:455]: Well, async-profiler in this version seems broken. I will switch the example to JFR instead which seems to be working fine.

Comment (adutra) [reply site/docs/guides/kubernetes.md]: Ready for review again.

**Review by dimas-b — APPROVED** [review]

File: site/docs/guides/kubernetes.md (hunk 1 above)
Comment (adutra) [reply site/docs/guides/kubernetes.md]: Ideally we could publish our own debug image with the right user (185), all the tools (CLI, admin) and all the utilities both for network and JVM diagnostics. But that's enough for now :-)

**Pull request merged**